src/geojsonm/geojsonm.ml

```diff
@@ -245,3 +245,42 @@ let fold_props f init src =
     | `Await -> assert false
   in
   try Ok (go init) with Abort e -> Error e
+
+let iter_geometry f src =
+  let decoder = Jsonm.decoder src in
+  let loc () = Jsonm.decoded_range decoder in
+  let rec go () =
+    match Jsonm.decode decoder with
+    | `Lexeme (`Name "geometry" as t) -> (
+        match G.Geometry.of_json @@ decode_single_object decoder with
+        | Error (`Msg m) -> raise (Abort (`Unexpected m))
+        | Ok g ->
+            let g' = f g in
```

Comment:
`f g` will be of type unit if I'm not mistaken, so `f g;` is sufficient (no need to bind it to a variable).

Reply:
Yeah my mistake , Thank you so much for pointing me this! :pray: